exchange/adapter_builders.go

```diff
@@ -234,6 +235,7 @@ func newAdapterBuilders() map[openrtb_ext.BidderName]adapters.Builder {
 		openrtb_ext.BidderAxis:              axis.Builder,
 		openrtb_ext.BidderAxonix:            axonix.Builder,
 		openrtb_ext.BidderBeachfront:        beachfront.Builder,
+		openrtb_ext.BidderBematterfull:      bematterfull.Builder,
 		openrtb_ext.BidderBeintoo:           beintoo.Builder,
```

Comment:
should maintain alpha sorted order here,  `openrtb_ext.BidderBematterfull` should be after 		`openrtb_ext.BidderBeintoo`

```
openrtb_ext.BidderBeintoo:           beintoo.Builder,
openrtb_ext.BidderBematterfull:      bematterfull.Builder,
```